src/main/java/essenmakanan/ui/Ui.java

```diff
@@ -131,9 +135,14 @@ public static void printStartRecipeMessage(IngredientList missingIngredients,
     }
 
     public static void printAllRecipes(RecipeList recipes) {
-        System.out.println("Here's a list of your recipes!");
-        recipes.listRecipeTitles();
-        drawDivider();
+        if (recipes.getRecipes().size() == 0) {
+            System.out.println("Your Recipe List is empty right now, please create your own recipe first :D!");
```

Comment:
Very concise handling of the first part of #151 ! However, did you miss out on error messages for when recipe list is empty and the user types view r/1? An error message should be printed in that case too.

Reply:
Will check later